src/bin/ion/commands/generate/utils.rs

```diff
@@ -127,6 +131,18 @@ impl Language for JavaLanguage {
             Template::Sequence => "sequence".to_string(),
         }
     }
+
+    fn namespace_separator() -> String {
```

Comment:
I think this could return `&'static str` since its value is always known at compile time?